velox/docs/configs.rst

```diff
@@ -206,6 +211,12 @@ Spilling
        If the limit is zero, then the spiller always spills a previously spilled partition if it has any data. This is
        to avoid spill from a partition with a small amount of data which might result in generating too many small
        spilled files.
+   * - spill_compression_codec
+     - string
+     - none
+     - Specifies the compression algorithm type to compress the spilled data before write to disk to trade cpu for
```

Comment:
This is unrelated change, right? Next time, please, consider making it in a separate PR.

"to trade cpu for spilling io efficiency" -> "to trade CPU for IO efficiency"